os/arch/xtensa/Kconfig

```diff
@@ -0,0 +1,83 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+if ARCH_XTENSA
+
+choice
+	prompt "XTENSA architecture selection"
+	default ARCH_CHIP_ESP32
+
+config ARCH_CHIP_ESP32
```

Comment:
We need to remove this entry also. It is good to add these ESP32 chip kconfig entries when  we add esp32 code.

Reply:
Oh, yes. It will be added later.